.circleci/config.yml

```diff
@@ -178,9 +194,17 @@ jobs:
   build_2.4.6_rails_5_1:
     docker:
       - image: ruby:2.4.6-alpine
+        environment:
```

Comment:
you could use a reusable block to cut down the copy/paste :) or if you update the version to 2.1, you can use commands: https://circleci.com/docs/2.0/reusing-config/

reusable block:
```yaml
common_ruby_environment: &common_ruby_environment
  environment:
    RAILS_ENV: test
    PGHOST: 127.0.0.1
    PGUSER: postgres
    POSTGRES_HOST_AUTH_METHOD: "trust"
common_pg_environment_vars: &common_pg_environment_vars
    POSTGRES_USER: postgres
    POSTGRES_DB: ror_testapp_test
    POSTGRES_HOST_AUTH_METHOD: "trust"
.
.
.
  build_2.4.6_rails_5_1:
    docker:	    docker:
      - image: ruby:2.4.6-alpine
       <<: *common_ruby_environment
      - image: postgres:9.6-alpine
         environment:
          PGDATA: /dev/shm/pgdata
          <<: *common_pg_environment_vars
```